api/openapi_server/_spec/openapi.yaml

```diff
@@ -420,30 +453,21 @@ paths:
           description: successful operation
       tags:
         - auth
-      x-openapi-router-controller: openapi_server.controllers.auth_controller
-      security:
-        - jwt:
-            - secret 
-  /auth/initialInvite:
+      x-openapi-router-controller: openapi_server.controllers.admin_controller
+  /auth/invite:
     post:
-      description: Removes auto generated password and replaces with user assigned password. Used for account setup.
-      operationId: initial_sign_in_reset_password
+      description: Invite user using AWS Congnito
+      operationId: invite
       requestBody:
         content:
           application/json:
             schema:
               type: object
               properties:
-                email:
-                  type: string
-                password:
-                  type: string
-                session:
+                username:
```

Comment:
Are we having them create a username only in sign up, or require it for sign in as well?

Reply:
@mira-kine thanks for pointing this out. It should be email instead of username. I'll switch it